authapi/views.py

```diff
@@ -83,7 +92,7 @@ def get_queryset(self):
         We also have the query params permission_contains and object_id, which
         allow users to filter the teams based on the permissions they
         contain.'''
-        queryset = self.queryset
+        queryset = super(TeamViewSetNoCreate, self).get_queryset()
         if self.action == 'list':
             archived = get_true_false_both(
                 self.request.query_params, 'archived', 'false')
```

Comment:
So this is kind of what we have, just with different names, and that `BaseTeamViewSet` == `OrganizationTeamViewSet`. The only difference between the nested viewset, and the root viewset, is that the nested viewset has a `create` action, where the root one does not, hench my `NoCreate` naming.